### ISSUE TITLE
Feature Request - Add support for black or white backgrounds around the circle

### ISSUE THREAD

**Issue by brkn** [TechForPalestine/profile-pic-maker#30]
Slack sucks with png images, it filled the transparent background with white.

**Comment by UmairJibran** [comment]
yep, that is a valid issue, slack added a white background to it, and it did not look good. I wonder if black would be any better.

What else can we do? Should we add black/white background ourselves, after a user toggles the transparent on/off?
